File: src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingDefendantRepository.java
package uk.gov.justice.probation.courtcaseservice.jpa.repository;

import org.springframework.data.repository.CrudRepository;
import uk.gov.justice.probation.courtcaseservice.jpa.entity.HearingDefendantEntity;

public interface HearingDefendantRepository extends CrudRepository<HearingDefendantEntity, Long> {
}

File: src/main/kotlin/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingDefendantRepository.kt
package uk.gov.justice.probation.courtcaseservice.jpa.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import uk.gov.justice.probation.courtcaseservice.jpa.entity.HearingDefendantEntity

@Repository
interface HearingDefendantRepository : CrudRepository<HearingDefendantEntity, Long> {
  fun findAllByDefendantCrn(crn: String): List<HearingDefendantEntity>
}
